bin/train_merge.sh

```diff
@@ -51,7 +51,7 @@ fi
 
 if [ -f "QAtrain_duo.C" ]; then
     echo "QAtrain_duo macro found"
-    if grep -q -E -e "/cpass1|cpass1/" *.xml; then
+    if grep -q -E -e "/cpass1|cpass1/" *.xml || grep -q -E -e "/cpass1|cpass1/" "$1"; then
```

Comment:
Does not affect Grid jobs, where input collection is always a XML file. It makes it work for text files too, in the release validation case (used if we don't put our files in zip archives).